### PR TITLE
ID and IK Tools throw for empty model filename

### PR DESCRIPTION
**Pull request by aseth1**
Added an explicit check for unspecified model file in `InverseDynamicsTool` and `InverseKinematicsTool` and to provide an appropriate error message.
Addresses #1099

### CODE REVIEW

File: OpenSim/Common/Object.cpp [hunk 1]
```diff
@@ -102,16 +102,13 @@ Object::Object(const string &aFileName, bool aUpdateFromXMLNode)
...
-            "Object: ERR- Empty filename encountered.";
-        throw Exception(msg,__FILE__,__LINE__);
-    } else 
-        if(!ifstream(aFileName.c_str(), ios_base::in).good()) {
-        string msg =
-            "Object: ERR- Could not open file " + aFileName+ ". It may not exist or you don't have permission to read it.";
-        throw Exception(msg,__FILE__,__LINE__);
-    }   
+    OPENSIM_THROW_IF(aFileName.empty(), Exception,
+        "Object::Cannot construct from empty filename. No filename specified.");
```
Comment (chrisdembia) [inline OpenSim/Common/Object.cpp]: The syntax `Object::` is misleading here; it makes me think that `Cannot` is a member of `Object`.

**Comment by chrisdembia** [comment]
Except for 1 comment, LGTM.

**Comment by tkuchida** [comment]
:+1: LGTM and @chrisdembia's concern was addressed. Merging.

**Comment by aseth1** [comment]
@chrisdembia @tkuchida thanks!

**Pull request merged**